src/types/payload.ts

```diff
@@ -33,7 +33,7 @@ export enum GitHubEvent {
 export enum UserType {
   User = "User",
   Bot = "Bot",
-  // Organization = "Organization",
+  Organization = "Organization",
```

Comment:
I thought this is a hallucination

Reply:
Yeah it was failing the payload validation